cypress/support/pages/course/pageEditCourse.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+class Edit_Course {
+  static #courseToEdit = '[aria-label="Kurs Mathe"]'
+  static #pageTitle = '[id="page-title"]'
+  static #dropDownCourse = '[class="btn btn-sm dropdown-toggle btn-course-dropdown"]'
+  static #btnCourseEdit = '[class="dropdown-item btn-course-edit"]'
+  static #selectClassesInput = '#classId_chosen > ul > li > input'
+  static #selectClassesDropdown = '#classId_chosen > div > ul > li:nth-child(1)'
+  static #selectClassesDeleteButtons = '#classId_chosen > ul > li > a'
+  static #selectClasses = '#classId_chosen > ul > li > span'
+  static #btnSubmit = '#main-content > section > form > div.modal-footer > button.btn.btn-primary.btn-submit'
+  static #titleEditPageString = 'Kurs bearbeiten'
+  static #classSearchString = 'ALLE'
+  static #numberStudentsInClass = 3
+  static #courseName = 'Mathe'
+  static #selectStudents = '#studentsId_chosen > ul > li'
+
+  clickOnCourse () {
+    cy.get(Edit_Course.#courseToEdit).click()
+  }
+
+  coursePageIsVisible () {
+    cy.url().should('include', '/courses')
+    cy.get(Edit_Course.#pageTitle).should('contain',Edit_Course.#courseName)
+  }
+
+  openCourseEditPage () {
+    cy.get(Edit_Course.#dropDownCourse).click()
+    cy.get(Edit_Course.#btnCourseEdit).click()
+  }
+
+  courseEditPageIsVisible () {
+    cy.get(Edit_Course.#pageTitle).should('contain',Edit_Course.#titleEditPageString)
+  }
+
+  selectClassForCourse () {
+    cy.get(Edit_Course.#selectClassesInput).type(Edit_Course.#classSearchString)
```

Comment:
Here (Edit_Course.#classSearchString), might make sense, since we use it twice, but I'd wait for the opinion of the others on this.